spk/squidguard/src/app/squidGuard.cgi

```diff
@@ -83,8 +83,8 @@ $autoinaddr  = 2;			# 0|1|2;
 					# 2 TO AUTORESOLVE & REDIRECT TO FIRST NAME
 
 # You may wish to enter your company link and logo to be displayed on the page
-my $company = "SynoCommunity";
-my $companylogo = "synocommunity.png";
+my $company = "http://SynoCommunity.com";
+my $companylogo = "http://$ENV{SERVER_NAME}:5000/webman/3rdparty/squidguard/synocommunity.png";
```

Comment:
the idea is good, but this variable cannot contain / and the path is "share/www/squidguardmgr"

--> not possible